index.html

```diff
@@ -0,0 +1,126 @@
+    <link rel="stylesheet" href="styles/home.css" />
+    <title>Friki Corner</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <div class="desktop flexrowDesk navsup">
+          <img src="Img/FaceIcon.png" alt="Facebook" />
+          <img src="Img/TwitterIcon.png" alt="Twitter" />
+          <ul class="ul flexrow">
+            <li>Logout</li>
+            <li>MyPage</li>
+            <li>Mexico</li>
+          </ul>
+        </div>
+        <div class="desktop flexrowDesk navdesk">
+          <a href="index.html"><img src="Img/logo.png" alt="Friki Corner" /></a>
+          <ul class="ul flexrow">
+            <li><a href="pages/about.html">About</a></li>
+            <li>Program</li>
+            <li>Join</li>
+            <li>Sponsor</li>
+            <li>News</li>
+            <li><button>Friki Corner</button></li>
+          </ul>
+        </div>
+        <div class="mobmenu flexrow">
+          <div class="mobile">
+            <img id="hambur" src="Img/mobile_menu.png" alt="Menu icon" />
+          </div>
+        </div>
+      </nav>
+      <div id="menu" class="menu">
+        <div class="flexcolum">
+          <img id="close" src="Img/CloseIcon.png" alt="Close" />
+          <ul class="ul flexcolum">
+            <li><a href="pages/about.html">About</a></li>
+            <li>Program</li>
+            <li>Join</li>
+            <li>Sponsor</li>
+            <li>News</li>
+          </ul>
+        </div>
+      </div>
+    </header>
+    <main>
+      <section class="heading">
+        <h3 class="margintop fontsec">"Hello frikies arround the world!"</h3>
+        <h1 class="margintop fontitul">Friki Corner <br />2023</h1>
+        <p class="description">
+          You welcome to the international convention about the leates releases in the comic world!
+          Come and meet people with the same likes and also enjoy the activities that we had
+          prepared for you. This amazing event will take place in December, in Mexico.
+        </p>
+        <h2>2023.12.17(sat) ~ 18(sun)</h2>
+        <p class="p">@Pepsi Center WTC, Mexico City</p>
+      </section>
+      <section class="program flexcolum">
+        <h3 class="subtit">Main Program</h3>
+        <div class="linea"></div>
+        <div class="mt">
+          <ul class="flexcolum ul flexrowDesk ul2">
+            <li class="flexrow flexcolumDesk">
+              <img src="Img/Program/contest.png" alt="Contest" />
+              <h4>Cosplay contest</h4>
+              <p>
+                Enroll or enjoy of the best cosplays in the contest where we have amazing prizes.
+              </p>
+            </li>
+            <li class="flexrow flexcolumDesk">
+              <img src="Img/Program/autographs.png" alt="autographs" />
+              <h4>Meet & Greet</h4>
+              <p>Meet your favorite voice actor or cartoonist and get an autograph and photo.</p>
+            </li>
+            <li class="flexrow flexcolumDesk">
```

Comment:
good job here as well, one small thing to add is the border hover effect to the cards in this section 

![image (5)](https://github.com/AndreaM2429/Capstone_project_Conference_page/assets/59707859/17e55cc2-2f98-4249-8e71-764b04a6f938)

Reply:
Done